crates/proof-of-sql/src/base/commitment/naive_commitment.rs

```diff
@@ -119,35 +119,37 @@ impl Commitment for NaiveCommitment {
                 let mut vectors: Vec<TestScalar> = vec![TestScalar::ZERO; offset];
                 let mut existing_scalars: Vec<TestScalar> = match cc {
                     CommittableColumn::Boolean(bool_vec) => {
-                        bool_vec.iter().map(|b| b.into()).collect()
+                        bool_vec.iter().map(std::convert::Into::into).collect()
                     }
                     CommittableColumn::TinyInt(tiny_int_vec) => {
-                        tiny_int_vec.iter().map(|b| b.into()).collect()
+                        tiny_int_vec.iter().map(std::convert::Into::into).collect()
                     }
                     CommittableColumn::SmallInt(small_int_vec) => {
-                        small_int_vec.iter().map(|b| b.into()).collect()
+                        small_int_vec.iter().map(std::convert::Into::into).collect()
+                    }
+                    CommittableColumn::Int(int_vec) => {
+                        int_vec.iter().map(std::convert::Into::into).collect()
                     }
-                    CommittableColumn::Int(int_vec) => int_vec.iter().map(|b| b.into()).collect(),
                     CommittableColumn::BigInt(big_int_vec) => {
-                        big_int_vec.iter().map(|b| b.into()).collect()
+                        big_int_vec.iter().map(std::convert::Into::into).collect()
                     }
                     CommittableColumn::Int128(int_128_vec) => {
-                        int_128_vec.iter().map(|b| b.into()).collect()
+                        int_128_vec.iter().map(std::convert::Into::into).collect()
                     }
                     CommittableColumn::Decimal75(_, _, u64_vec) => {
-                        u64_vec.iter().map(|b| b.into()).collect()
+                        u64_vec.iter().map(std::convert::Into::into).collect()
```

Comment:
We may as well import `core::convert::Into`. Also please bear in mind that we need to support `no_std` so unless it is absolutely necessary let's not use `std`.